### ISSUE TITLE
Support for .net MAUI projects.

### ISSUE THREAD

**Issue by Luckre** [Vungle/Windows-SDK#47]
Hi developers,

Is it possible to make a compatible .net 8 SDK for .net MAUI.
I am converting all my previous projects from Xamarin to .net MAUI. I tried to make native bindings for ios from the framework SDK but it failed to initialize.

Is there is a future plan to add support for .net maui or even the standard .net 8 ios, android and windows?

Thank you

**Comment by gregthatcher** [comment]
Yes, we are working on a Maui SDK, but are still some months away from having a working Beta version.  Would you like me to put you on our Beta testers list?

**Comment by Luckre** [comment]
Yes please. I'm glad to join.

I tried to create binding library for .net 8 iOS but I didn't release it. I can put it on github and share it with you if you like.

**Comment by gregthatcher** [comment]
Thanks much @Luckre , I have added you to the list.  Yes, if you can post some code that might help us out.

**Comment by KGLongWang** [comment]
SDK for QT?

**Comment by gregthatcher** [comment]
Sorry, but we do not currently have any plans to support the QT cross-platform library (https://www.qt.io/).

**Comment by Moreno-Borsalino** [comment]
It's very interesting that there are plans to make a MAUI version. I have a MAUI Android app based on .net 10 and would like to join the beta testing. Is that still possible?

**Comment by gregthatcher** [comment]
Unfortunately, we are way behind on our Maui development.

We _do_ have some other new SDKs that you might find useful.
https://www.nuget.org/packages/VungleSDKForWindowsDesktopNative/ is for C++/Win32

https://www.nuget.org/packages/VungleSDKForWinForms/ is for WinForms, and

https://www.nuget.org/packages/VungleSDKForWinUI3/ is for WinUI 3.

Thanks,
Greg

**Comment by Moreno-Borsalino** [comment]
Thanks, Greg, for your quick response.
It's a shame the MAUI version isn't ready yet, but I encourage you to continue your efforts to advance this project, which I'm sure will interest many people.
Meanwhile, for another ongoing project, I'm about to implement your SDK for WINUI 3, and I really hope it works.
Thanks,
Moreno
